pkg/agent/plugin/workloadattestor/k8s/k8s.go

```diff
@@ -142,17 +142,32 @@ type ExperimentalK8SConfig struct {
 
 // SigstoreHCLConfig holds the sigstore configuration parsed from HCL
 type SigstoreHCLConfig struct {
-	// EnforceSCT is the parameter to be set as false in case of a private deployment not using the public CT
-	EnforceSCT *bool `hcl:"enforce_sct, omitempty"`
+	// AllowedIdentities is a list of identities (issuer and subjects) that must match for the signature to be valid.
+	AllowedIdentities map[string][]string `hcl:"allowed_identities"`
```

Comment:
all those configuration changes are going to break compatibility, 
We may be OK with that since it is an experimental feature.
The same will happens with all selector changes.

Reply:
Yes, these changes introduce breaking changes. The goal is to simplify and improve the configuration options and selectors. Since this is an experimental feature, I believe these changes are acceptable and will ultimately enhance usability and functionality.